File: Modul5/Bahasa_Python/PRAK505-2310817210012-Nazmi_Hakim.py
def biodata(tahun_lahir, nama, asal):
    tahun_sekarang = 2023
    usia = tahun_sekarang - tahun_lahir

    print("Perkenalkan Nama Saya :", nama)
    print("Umur Saya :", usia)
    print("Saya Adalah Angkatan :", tahun_sekarang)
    print("Asal Saya dari :", asal)

if __name__ == "__main__":
    tahun_lahir = int(input())
    nama = input()
    asal = input()

    biodata(tahun_lahir, nama, asal)

File: Modul5/Bahasa_C/PRAK505-2310817210012-Nazmi_Hakim.c
#include <stdio.h>
void Biodata(int A, char B[], char C[]){
int tahun_sekarang = 2023;
int usia = tahun_sekarang - A;

printf("Perkenalkan Nama Saya : %s\n", B);
printf("Umur Saya : %d\n", usia);
printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
printf("Asal Saya dari : %s\n", C);
}
int main() {
int tahunLahir;
 char Namaku[20], Asal[15];
 scanf(" %d",&tahunLahir);
 scanf(" %[^\n]%*c",&Namaku);
 scanf(" %[^\n]%*c",&Asal);
Biodata(tahunLahir, Namaku, Asal);
return 0;
}